lib/logproto/logproto-builtins.c

```diff
@@ -24,7 +24,6 @@
 #include "logproto-dgram-server.h"
```

Comment:
Add OneIdentity copyright to the file.
The correct copyright today is: `Copyright (c) 2024 One Identity LLC.`